Lesson4/Task2/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Text.RegularExpressions;
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter URL: ");
+            string url = Console.ReadLine();
+
+            WebRequest request = WebRequest.Create(url);
```

Comment:
`HttpClient` is a better choice when working with http requests and responses 
see: https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/console-webapiclient#making-web-requests (but it contains some still not covered information as `async` and `await` keywords)